package.json

```diff
@@ -16,9 +16,10 @@
     "eslint": "eslint src --ext .js",
     "eslint:fix": "eslint src .js --fix",
     "travis": "npm run build:test",
-    "build:demo": "cd demo && npm run build",
-    "build:demo-v1": "cd demo-v1 && npm run build",
+    "build:demo": "cd demo && npm i && npm run build",
```

Comment:
Might be good practice to use the [npm ci](https://docs.npmjs.com/cli/v9/commands/npm-ci) command in automated environments like github actions.